lib/ae_mdw/sync/async_tasks/update_aex9_presence.ex

```diff
@@ -1,6 +1,6 @@
-defmodule AeMdw.Sync.AsyncTasks.UpdateAex9Presence do
+defmodule AeMdw.Sync.AsyncTasks.UpdateAex9State do
```

Comment:
filename not renamed with `git mv` as by the change percentage it's creating a new one.